src/applications/coronavirus-screener/components/FormResult.jsx

```diff
@@ -43,17 +46,33 @@ export default function FormResult({ formState }) {
   );
 
   function recordScreeningToolEvent(screeningToolResult) {
-    if (!resultSubmitted) {
+    if (!resultSubmitted.isSubmitted) {
+      const timeToComplete = moment.duration(
+        moment().diff(resultSubmitted.startTime),
+      );
+      const formattedTimeToComplete = [
+        timeToComplete.hours(),
+        timeToComplete.minutes(),
+        timeToComplete.seconds(),
+      ]
+        .map(segment => segment.toString().padStart(2, '0'))
+        .join(':');
```

Comment:
this should also work:
```js
moment.utc(timeToComplete.as('milliseconds')).format('HH:mm:ss')`
```
the only downside is if its >24hr it will loop back to 0